apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrdersTable.tsx

```diff
@@ -187,14 +178,22 @@ export function OrdersTable({
     })
   }, [tableHeaders, currentTab])
 
+  // Determine if this is a TWAP table by checking if any order has composableCowInfo with a parentId
+  const isTwapTable = useMemo(() => {
```

Comment:
😄 Thats a tricky solution.
Mainly, we should not do like that. Pure components must be as simple as stupid.
In this case `isTwapTable` should be passed as a prop in this component.